### ISSUE TITLE
NTR: Argininosuccinate synthetase activity low in fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9066]
New term request Argininosuccinate synthetase activity low in fibroblasts (f407): ASS

**Comment by pnrobinson** [comment]
@MickeySegal
this seems to be mire highly expressed in the liver -- https://en.wikipedia.org/wiki/Argininosuccinate_synthase
perhaps it would be best to mention this in the definition but not in the term label?

**Comment by MickeySegal** [comment]
It is easier to do a skin biopsy, so that is likely why we defaulted to fibroblasts.  But I agree that the finding doesn't need to specify the tissue in the title, and one could explain that it can be determined from tissues such as liver or skin.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000353
- label: Reduced tissue argininosuccinate synthetase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:28111830
- def: Concentration of argininosuccinate synthetase (EC 6.3.4.5) in body tissues is below the lower limit of normal. Clinically, argininosuccinate synthetase is measured in the liver or in cultered skin fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9066
- synonyms: Reduced argininosuccinate synthetase activity in cultured fibroblasts(NONE)|Reduced hepatic argininosuccinate synthetase activity(NONE)
